module/VuFind/src/VuFind/ILS/Driver/Symphony.php

```diff
@@ -417,8 +417,8 @@ protected function getStatuses999Holdings($ids)
                 $available = (empty($curr_loc) || $curr_loc == $home_loc)
                     || $result['circulate flag'] == 'Y';
                 $callnumber = $result['call number'];
-                $location   = $library . ' - ' . ($available && !empty($curr_loc)
-                    ? $curr_loc : $home_loc);
+                $location   = $library . ' - ' .
+                    ($available ? $curr_loc : $home_loc);
```

Comment:
I'm still not sure I understand the motivation of this change. Isn't it still possible that $available could be true because $result['circulate_flag'] is 'Y' yet $curr_loc could be empty (say, because $result['current location'] was set to a blank string)? In this case, wouldn't we still want to fall back on $home_loc?

Of course, I am not familiar with Symfony or this code, so I don't know realistically what is likely or unlikely... but just looking at the code in a purely abstract way, this feels like a potential behavior change.

Reply:
Looking a bit more into it I think you are right, I changed it back.